examples/ding-dong-wechaty-bot.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Wechaty - WeChat Bot SDK for Personal Account, Powered by TypeScript, Docker, and 💖
+ *  - https://github.com/chatie/wechaty
+ */
+import {
+  Message,
+  Wechaty,
+  log,
+  FileBox,
+}               from 'wechaty'
```

Comment:
We should not use any `wechaty` dependency in our puppet repo because that will cause problems.

Please just use the raw puppet API to write the example, and keep only one file with the name `ding-dong-bot.ts`.

Thanks!